EscapeVR/src/main/java/nl/tudelft/kroket/input/interaction/MovementHandler.java

```diff
@@ -74,18 +76,17 @@ public void setLockHorizontal(boolean lockHorizontal) {
   public void onAction(String name, boolean keyPressed, float tpf) {
```

Comment:
It's an Override. 

```
Called when an input to which this listener is registered to is invoked.

Specified by: onAction(...) in ActionListener

Parameters:name The name of the mapping that was invoked
keyPressed True if the action is "pressed", false otherwise
tpf The time per frame value.
```